CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
-# [TBD] - TBD
+# TBD
+## Added
+* MetastoreUnavailableException and MetaStoreUriNormaliser implementations.
```

Comment:
I guess it kind of looks weird to have classes here that aren't actually used anywhere else in the package. Generally we only do this if we have >1 usage downstream. Having said that, they're small and generally low functionality and easy to understand and you've already done the work. I'll approve this but if someone else really feels strongly about them not being moved here that's fine by me too.